src/tests/SelectedRevisionsTable.test.tsx

```diff
@@ -16,17 +18,16 @@ describe('Search View', () => {
 
   it('should delete revisions after click and not show revisions table if no revisions', async () => {
     const dispatchSpy = jest.spyOn(store, 'dispatch');
+    const user = userEvent.setup({ delay: null });
 
-    const { rerender } = render(<SelectedRevisionsTable />);
```

Comment:
I was unsure what the purpose of this was, so I tried removing it and the test still worked as expected. If there's justification for keeping this I can revert this change.